roles/kafka-mesos/tasks/destroy.yml

```diff
@@ -19,7 +20,7 @@
     - kafka-mesos
 
 - name: Remove all Kafka brockers
-  when: existing_brockers | int > 0
+  when: existing_brockers.stdout | int > 0
```

Comment:
It's bug obviously, thanks you for suggestion but nevertheless, code should look like that -> existing_brockers > 0, we don't need to convert output into integer since we don't want to do any math.